docs/concepts/workloads/vm.md

```diff
@@ -438,3 +438,9 @@ The value of `spec.biosUUID` is used to set the vSphere VM `config.uuid` field,
 ``` bash
 govc vm.info -vm.uuid $(k get vm -o jsonpath='{.spec.biosUUID}' -n $ns $name)
 ```
+
+## Guest IDs
+
+The `spec.guestID` is an optional field that can be used to specify the guest operating system identifier when creating a new VM or when the existing VM is powered off.
+A complete list of supported guest IDs can be found [here](https://vdc-repo.vmware.com/vmwb-repository/dcr-public/e1580753-930f-4c00-9105-77dd2e8913f8/98ce8e90-a51b-4e41-8dac-3e8fd5409f2b/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html).
```

Comment:
Same here (U1 vs U2). Maybe the govc example could be included here too. Or even use it generate a list we could add to these docs?

Reply:
Updated the link and also added the govc command example.